.gitignore

```diff
@@ -77,6 +77,7 @@
 !/plugins/expressionQuestionHelp/
 !/plugins/expressionFixedDbVar/
 !/plugins/expressionQuestionForAll/
+!/plugins/statFunctions/
```

Comment:
Is "stat" short for "static"? Or "statistics"?

Reply:
statistics, can rename

Reply:
Maybe statisticsCountFunction if rename

Reply:
But answers are being counted? "AnswerCountFunctions"?

Reply:
Yes, but can add statFunctions::medium(…) after …

AnswerCountFunctions : seems more related to Count answers , it's Reponse count …

Reply:
PS : currently \statFunctions\countFunctions::statCount + \statFunctions\countFunctions::statCountIf , but can add

\statFunctions\mediumFunctions::… or … or … or …